### PR TITLE
Ipv6 any.proxy fixes

### PR DESCRIPTION
**Pull request by BenGardiner**
changes to get any.proxy working in a IPV6-only & link-local-only setup

### CODE REVIEW

File: modules/any_proxy/any_proxy.go [hunk 1]
```diff
@@ -43,8 +43,8 @@ func NewAnyProxy(s *session.Session) *AnyProxy {
 		"Leave empty to intercept any source address."))
 
 	mod.AddParam(session.NewStringParameter("any.proxy.dst_address",
-		session.ParamIfaceAddress,
```
Comment (evilsocket) [inline modules/any_proxy/any_proxy.go:46]: hey @BenGardiner can't we just leave the IPv4 default? it's just more convenient for the vast majority of the cases ... what do you think?

Comment (BenGardiner) [reply modules/any_proxy/any_proxy.go:46]: yup @evilsocket , good call. I think the important change was removing the IPV4 validator and this was collateral damage when I was editing.

Comment (evilsocket) [reply modules/any_proxy/any_proxy.go:46]: great! can you update the PR or do you prefer if I do it post merge?

Comment (BenGardiner) [reply modules/any_proxy/any_proxy.go:46]: I did an online edit to the branch. hopefully that didn't break anything

**Review by evilsocket — APPROVED** [review]

File: modules/any_proxy/any_proxy.go (hunk 1 above)
Comment (evilsocket) [reply modules/any_proxy/any_proxy.go:46]: thank you very much! merging 👍🏻

**Pull request merged**